h2o-genmodel/src/main/java/hex/genmodel/algos/tree/SharedTreeMojoReader.java

```diff
@@ -15,15 +13,8 @@ protected void readModelData() throws IOException {
     // In mojos v=1.0 this info wasn't saved.
     Integer tpc = readkv("n_trees_per_class");
     if (tpc == null) {
-      if (_model instanceof DrfMojoModel) {
-        tpc = ((DrfMojoModel) _model)._effective_n_classes;
-      }
-      else if (_model instanceof GbmMojoModel) {
-        tpc = ((GbmMojoModel) _model).calcNClassesToScore();
-      }
-      else {
-        throw new RuntimeException("Unknown Mojo Model");
-      }
+      Boolean bdt = readkv("binomial_double_trees");  // This flag exists only for DRF models
```

Comment:
As you may notice, the old code contained a bug: the `tpc = ((DrfMojoModel) _model)._effective_n_classes;` was executing at the moment when `DrfMojoReader` hadn't read the model information yet, and therefore `_effective_n_classes` will be 0.

We didn't keep any old DRF models around (before the `n_trees_per_class` variable was added), so this error was not caught ...